ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -224,9 +217,12 @@ export default class ConfirmPageContainerContent extends Component {
                       <Button
                         type="inline"
                         className="confirm-page-container-content__link"
-                        onClick={() =>
-                          this.setState({ setShowDepositPopover: true })
-                        }
+                        onClick={() => {
+                          const portfolioUrl = process.env.PORTFOLIO_URL;
+                          global.platform.openTab({
+                            url: `${portfolioUrl}/buy?metamaskEntry=ext_buy_button`,
+                          });
+                        }}
```

Comment:
Hey @alaahd : we are working for increasing test coverage of confirmation pages, it will be nice to have above piece also tested. `global.platform.openTab` can probably be mocked for writing test.